assets/javascripts/kitten/components/cards/contribution-card.js

```diff
@@ -0,0 +1,461 @@
+      src: '',
+      alt: '',
+    },
+    titleAmount: '',
+    titleDescription: '',
+    textDescription: '',
+  }
+
+  render() {
+    // We need to destructure the props to prevent them to hydrate children components.
+    const {
+      isDisabled,
+      viewportIsSOrLess,
+      viewportIsMobile,
+      viewportIsTabletOrLess,
+      titleAmount,
+      titleDescription,
+      textDescription,
+      titleContributors,
+      titleSmallContributors,
+      titleDelivery,
+      titleSmallDelivery,
+      titleAvailability,
+      titleSmallAvailability,
+      valueContributors,
+      valueDelivery,
+      valueAvailability,
+      button,
+      buttonOnMouseEnter,
+      buttonOnMouseLeave,
+      buttonOnClick,
+      myContribution,
+      manageContribution,
+      manageContributionLink,
+      imageProps,
+      ...others
+    } = this.props
+
+    const styleCard = [
+      others.style,
+      styles.card,
+      isDisabled && styles.card.isDisabled,
+    ]
+
+    return (
+      <StyleRoot {...others} style={styleCard}>
+        <Marger bottom={viewportIsSOrLess ? 0 : 5}>
+          <Grid style={styles.card.addPadding} disabled={isDisabled}>
+            <GridCol col-l="7" col-m={!imageProps.src ? 10 : 7}>
+              {this.renderDescription()}
+            </GridCol>
+            {imageProps.src && (
+              <GridCol col-m="4" offset-m="1">
+                {this.renderImage()}
+              </GridCol>
+            )}
+          </Grid>
+          {viewportIsSOrLess && this.renderChoiceButton()}
+        </Marger>
+      </StyleRoot>
+    )
+  }
+
+  renderDescription() {
+    const {
+      button,
+      titleAmount,
+      titleDescription,
+      textDescription,
+      tag,
+      viewportIsSOrLess,
+      viewportIsMobile,
+    } = this.props
+
+    return (
+      <Fragment>
+        <Marger top={viewportIsSOrLess ? 4 : 5}>
+          <Marger bottom="2">
+            <Title
+              modifier={viewportIsSOrLess ? 'tertiary' : 'secondary'}
+              italic
+              margin={false}
+              tag={tag}
+            >
+              {titleAmount}
+            </Title>
+          </Marger>
+          <Marger top="2" bottom={viewportIsMobile ? 3 : 4}>
+            <HorizontalStroke size="big" />
+          </Marger>
+          {titleDescription && (
+            <Marger top="4" bottom="1">
+              <Text
+                size={viewportIsSOrLess ? 'big' : 'huge'}
+                tag={tag}
+                weight="bold"
+              >
+                {titleDescription}
+              </Text>
+            </Marger>
+          )}
+          <Marger top={!titleDescription ? 4 : 1}>
+            <Paragraph
+              modifier={viewportIsSOrLess ? 'quaternary' : 'tertiary'}
+              margin={false}
+            >
+              {textDescription}
+            </Paragraph>
+          </Marger>
+        </Marger>
+        {this.props.render ? this.props.render() : null}
```

Comment:
Peut être remplacer par `{this.props.render && this.props.render()}`. Le problème étant que `this.props.render` peut s'afficher vu que c'est pas un boolean. Il faut donc utiliser `!!this.props.render`. Ce qui donne : 

`{ !!this.props.render && this.props.render() }`